doc/nrf/device_guides/working_with_nrf/nrf54h/features_nrf54h20.rst

```diff
@@ -0,0 +1,181 @@
+
+Power management service
+========================
+
+Rather than manually setting a power mode through a register or calling a sleep instruction, power management is performed through a service where the application describes the functionality it needs and how much latency it can accept.
+Based on this information, the Power Management Service automatically handles the details such as: deciding how registers will be retained, which parts of the device are put into what mode, and what clock signals are running.
+
+Security architecture
+=====================
+
+The nRF54H20 SoC security architecture is based on the conclusions from extensive threat modelling.
+nRF54H20 is designed to protect against logical attacks and firmware-oriented attacks, as well as physical security at the PCB or package level (for example, against voltage glitching and EMI attacks).
+However, robustness against physical die probing or alteration of the die itself is not provided by the design.
+Security in software is implemented as a set of services, provided by the Secure Domain.
+These services manage the following:
+
+* Secure Boot
+* Cryptographic operations
+* Key management
+* Device Firmware Updates (DFU) through the Software Internet of Things (SUIT) procedure
+
+The main application and other parts of the system access security services through an API.
+This API is translated internally to messages sent to the Secure Domain through a mailbox-based interface.
+The security services handle these requests, utilizing hardware accelerators and other resources, and as security is centralized in the Secure Domain, keys can be kept inside this physical area and remain well-protected against attacks or breaches due to firmware exploits.
+The nRF54H20 SoC also includes tamper sensors that can detect an attack in progress and take appropriate action.
+The cryptographic accelerators have been hardened against side-channel attacks.
+
+CPUs overview and features
+==========================
+
+The CPU cores in the nRF54H20 SoC are based on two types of CPU architectures:
+
+* Arm Cortex M33
+* RISC-V (VPR cores)
+
+Arm Cortex M33
+--------------
+
+The Arm Cortex-M33 is a powerful yet power-efficient CPU developed by Arm.
+It supports TrustZone, allowing secure and non-secure code to be separated and to have differentiated access to memory and peripherals.
+In the nRF54H20 SoC, both Cortex-M33 instances are configured with a floating point unit, TrustZone, and DSP extensions enabled.
+In addition to the CPU itself, each Arm Cortex-M33-based core includes also the following local features:
+
+* Local level-1 (L1) cache, split into separate caches for data and instructions, with 16 kB each[2].
+* Local RAM, serving as Tightly Coupled Memory (TCM), accessible without any wait states or latency.
+* A Memory Vector DMA (MVDMA) controller, which allows high-throughput DMA transfers between local and global memory.
```

Comment:
Glossary has Memory-to-Memory Vector DMA. Same or different? If same, should probably be a term.